portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,30 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
```

Comment:
Please keep sorted: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing